domains/maskduck.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "MaskDuck's personal website!",
+  "repo": "https://github.com/continental-beauty-msbacdau/continental-beauty-msbacdau.github.io",
+  "owner": {
+    "github": "continental-beauty-msbacdau",
```

Comment:
change the `github` property to `username`

Reply:
@TheBotlyNoob, you mean change `github` into `username`?